src/bitcoin-cli.cpp

```diff
@@ -569,6 +569,56 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "> bitcoin-cli -netinfo help\n"};
 };
 
+/** Process addressinfo requests */
+class AddressinfoRequestHandler : public BaseRequestHandler
+{
+private:
+    static constexpr uint8_t m_networks_size{5};
+    const std::array<std::string, m_networks_size> m_networks{{"ipv4", "ipv6", "torv2", "torv3", "i2p"}};
```

Comment:
57eb71ce0f27c0353ea0742a696c6d04ce048e15

I think you can omit template arguments, see https://stackoverflow.com/a/61371266/1978589.

Reply:
Thanks, will give it a try in the next push.

Reply:
Couldn't get that to compile (clang 11).